docs/lease-metrics.md

```diff
@@ -0,0 +1,8 @@
+# Leases Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| kube_lease_created | Gauge | `lease`=&lt;lease-name&gt; | EXPERIMENTAL |
```

Comment:
Can you explain what this metric is useful for?

Reply:
This metric will be useful to know when this object was created (the node was added to the cluster).

Reply:
That makes sense but do you have a concrete use in an alerting rule for example? I get how renew and owner can make a useful alert with rich information, but I don’t see it with this metric.

Reply:
I agree with you, this metric looks redundant. I removed it.